plugins/MIDIPressureToCC/PluginMIDIPressureToCC.cpp

```diff
@@ -0,0 +1,214 @@
+    : Plugin(paramCount, presetCount, 0)  // 0 states
+{
+    loadProgram(0);
+}
+
+// -----------------------------------------------------------------------
+// Init
+
+void PluginMIDIPressureToCC::initParameter(uint32_t index, Parameter& parameter) {
+    if (index >= paramCount)
+        return;
+
+    parameter.hints = kParameterIsAutomable | kParameterIsInteger;
+    parameter.ranges.def = 0;
+    parameter.ranges.min = 0;
+    parameter.ranges.max = 127;
+
+    switch (index) {
+        case paramFilterChannel:
+            parameter.name = "Filter Channel";
+            parameter.symbol = "channelf";
+            parameter.ranges.def = 0;
+            parameter.ranges.min = 0;
+            parameter.ranges.max = 16;
+            parameter.enumValues.count = 17;
+            parameter.enumValues.restrictedMode = true;
+            {
+                ParameterEnumerationValue* const channels = new ParameterEnumerationValue[17];
+                parameter.enumValues.values = channels;
+                channels[0].label = "Any";
+                channels[0].value = 0;
+                channels[1].label = "Channel 1";
+                channels[1].value = 1;
+                channels[2].label = "Channel 2";
+                channels[2].value = 2;
+                channels[3].label = "Channel 3";
+                channels[3].value = 3;
+                channels[4].label = "Channel 4";
+                channels[4].value = 4;
+                channels[5].label = "Channel 5";
+                channels[5].value = 5;
+                channels[6].label = "Channel 6";
+                channels[6].value = 6;
+                channels[7].label = "Channel 7";
+                channels[7].value = 7;
+                channels[8].label = "Channel 8";
+                channels[8].value = 8;
+                channels[9].label = "Channel 9";
+                channels[9].value = 9;
+                channels[10].label = "Channel 10";
+                channels[10].value = 10;
+                channels[11].label = "Channel 11";
+                channels[11].value = 11;
+                channels[12].label = "Channel 12";
+                channels[12].value = 12;
+                channels[13].label = "Channel 13";
+                channels[13].value = 13;
+                channels[14].label = "Channel 14";
+                channels[14].value = 14;
+                channels[15].label = "Channel 15";
+                channels[15].value = 15;
+                channels[16].label = "Channel 16";
+                channels[16].value = 16;
+            }
+            break;
+        case paramKeepOriginal:
+            parameter.name = "Keep original Pressure events";
+            parameter.shortName = "Keep original";
+            parameter.symbol = "keep_original";
+            parameter.hints |= kParameterIsBoolean;
+            parameter.ranges.max = 1;
+            break;
+        case paramCC:
+            parameter.name = "Target CC";
```

Comment:
Please call this "Destination CC", as I have another plugin coming up, where I use that term.